shared/reducers/tracker.js

```diff
@@ -604,14 +588,17 @@ function updateProof (proofs: Array<Proof>, rp: RemoteProof, lcr: LinkCheckResul
   return updated
 }
 
-function deriveTrackerState (
-  allOk: boolean,
-  anyWarnings: boolean,
-  anyError: boolean,
-  anyPending: boolean,
-  anyDeletedProofs : boolean,
-  anyUnreachableProofs : boolean,
-  eldestKidChanged: boolean
+export function overviewStateOfProofs (proofs: Array<Proof>): OverviewProofState {
```

Comment:
I agree. I split this up initially because I assumed it was being used in other places to arrive tracker state, I didn't recognize that everything was accessing the tracker section of the store. I think the abstraction does bring some clarity to what's going on, which makes it worthwhile, but I agree that there should be more uses of it to fully justify the split.